samples/zigbee/template/CMakeLists.txt

```diff
@@ -0,0 +1,17 @@
+#
+# Copyright (c) 2021 Nordic Semiconductor ASA
+#
+# SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+#
+
+cmake_minimum_required(VERSION 3.13.1)
```

Comment:
`3.20.0` as per https://github.com/nrfconnect/sdk-nrf/pull/5262

Reply:
Currently it fails the build:
```
CMake Error at CMakeLists.txt:7 (cmake_minimum_required):
  CMake 3.20.0 or higher is required.  You are running version 3.17.3
```

Reply:
@tejlmand Is this being covered with regards to #5262? It seems to be an issue with our build servers running older CMake.

Reply:
I'll give it the last chance. If it fails - I am going to revert it to the 3.17.3. It is counterproductive to wait for this PR (we are waiting for it for 7 days now and the CI is still red there).

Reply:
Reverted.